helm/alfresco-content-services/templates/config-repository.yaml

```diff
@@ -40,13 +40,19 @@ data:
       {{- if index .Values "alfresco-search" "enabled" }}
       -Dsolr.host={{ template "alfresco-search.host" . }}
       -Dsolr.port={{ template "alfresco-search.port" . }}
+      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "none" }}
       {{- else }}
       {{- if index .Values "alfresco-search" "external" }}
       -Dsolr.host={{ index .Values "alfresco-search" "external" "host" }}
       -Dsolr.port={{ index .Values "alfresco-search" "external" "port" }}
+      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "none" }}
       {{- end }}
       {{- end }}
-      -Dsolr.secureComms={{ .Values.global.tracking.auth | default "none" }}
+      {{- if index .Values "alfresco-elasticsearch-connector" "enabled" }}
```

Comment:
do I need here to start using standard camelCase naming convention and avoid using lowercase-separated-with-dash?

Reply:
We might regret it at some point but let's stick to dash notation for now...